hotham/src/rendering/camera.rs

```diff
@@ -6,41 +6,41 @@ use crate::util::posef_to_isometry;
 #[derive(Debug, Clone)]
 /// The Camera, or View, in a scene.
 pub struct Camera {
-    /// The camera's position in space
-    pub position: Isometry3<f32>,
+    /// The camera's position in globally oriented stage space
+    pub gos_from_view: Isometry3<f32>,
     /// The view matrix
-    pub view_matrix: Matrix4<f32>,
+    pub view_from_gos: Matrix4<f32>,
 }
 
 impl Default for Camera {
     fn default() -> Self {
         let t = Translation3::from(Vector3::zeros());
         Self {
-            position: Isometry3::from_parts(t, UnitQuaternion::identity()),
-            view_matrix: Matrix4::identity(),
+            gos_from_view: Isometry3::from_parts(t, UnitQuaternion::identity()),
+            view_from_gos: Matrix4::identity(),
         }
     }
 }
 
 impl Camera {
-    /// Update the camera's position from an OpenXR view
-    pub fn update(&mut self, view: &View) -> Matrix4<f32> {
+    /// Update the camera's position from an OpenXR view (stage from view)
```

Comment:
This comment is a bit unclear; I'm not sure it's necessary to mention the stage here.